app/src/main/java/org/mozilla/fenix/compose/Divider.kt

```diff
@@ -0,0 +1,154 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Generic Divider
+ *
+ * @param modifier [Modifier] Used to shape and position the underlying [androidx.compose.material.Divider]
+ * @param color  [Color] of the divider line
+ */
+@Composable
+fun Divider(
+    modifier: Modifier = Modifier,
+    color: Color = FirefoxTheme.colors.borderPrimary,
```

Comment:
I don't feel strongly about this, but I am wondering if we should just inline the `FirefoxTheme.colors.borderPrimary` color for now and not provide a param for modifying the `color`, the intention here is to be a bit more restrictive and opinionated initially with the API that we expose and loosen it slowly once we see valid use cases that comes up.

Reply:
ok, sounds good, I'll remove the param for modifiying `color`